modules/core/src/controllers/controller.ts

```diff
@@ -503,19 +509,18 @@ export default class Controller {
   }
 
   // Default handler for the `wheel` event.
-  _onWheel(event) {
+  protected _onWheel(event: MjolnirWheelEvent): boolean {
```

Comment:
Nit: Too late to change this now, but I am not very keen on prefixing symbols with framework names. Normally we control such things by explicitly importing what we need. If a prefix is needed I prefer to make it more semantic prefix rather than slap on a framework name. And importing apps can add the Mjolnir prefix if they need it. But I can see why we did it this way...

Reply:
Typed mjolnir.js is still in alpha :)